superwall/src/main/java/com/superwall/sdk/network/device/Capability.kt

```diff
@@ -0,0 +1,35 @@
+package com.superwall.sdk.network.device
+
+import com.superwall.sdk.analytics.superwall.SuperwallEvents
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.encodeToJsonElement
+
+@Serializable
+internal sealed class Capability(
+    @SerialName("name")
+    val name: String,
+) {
+    @Serializable
+    @SerialName("paywall_event_receiver")
+    class PaywallEventReceiver : Capability("paywall_event_receiver") {
+        @SerialName("event_names")
+        val eventNames =
+            listOf(
+                SuperwallEvents.TransactionStart,
```

Comment:
Need to add in restore events here, or are they not merged yet?

Reply:
Ah I was wondering why they don't exist, haven't added it to `SuperwallEvents` enum just to the trackable sealed class. Fixed that now and added them to the enum and here